File: A/A1420.java
import java.util.Scanner;

public class A1420 {
    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);
        int t = input.nextInt();
        boolean done;
        int n;
        long x,last;
        while (t-- != 0) {
            n = input.nextInt();
            last = -1;
            done = false;
            for(int i=0;i<n;i++){
                x = input.nextLong();
                if(i > 0 && x >= last){
                    done = true;
                }
                last = x;
            }
            System.out.println(done ? "YES" : "NO");
        }
        input.close();
    }
}

File: A/A1420.cpp
#include<iostream>
#include<vector>
#include<algorithm>
using namespace std;
int main(){
    long last;
    bool a;
    int t,n;
    cin>>t;
    while(t--){cin>>n;last = -1;a=false;vector<long> v(n); for(int i=0;i<n;i++){cin>>v[i];if(i > 0 && v[i] >= last){a=true;} last = v[i];}puts(a ? "YES":"NO");}
    return 0;
}